fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -368,6 +380,42 @@ public FedoraResource getContainer() {
     }
 
     @Override
+    public FedoraResource getLDPCv() {
+        try {
+            return Optional.of(getNode().getNode(LDPCV_TIME_MAP)).map(nodeConverter::convert).orElse(null);
+        } catch (PathNotFoundException e) {
+            throw new PathNotFoundRuntimeException(e);
+        } catch (RepositoryException e) {
+            throw new RepositoryRuntimeException(e);
+        }
+    }
+
+    @Override
+    public FedoraResource findOrCreateLDPCv() {
+        final Node ldpcvNode;
+        try {
+            ldpcvNode = findOrCreateChild(getNode(), LDPCV_TIME_MAP, NT_FOLDER);
```

Comment:
Is this making the LDPCv a childe of the LDPR?  If not, where does the LDPCv live?  

To be clear, the reason I'm wondering is that it's explicit in the spec that the LDPCv cannot be a child of the LDPR.

Reply:
at least via the ldp:contains triple - it could be part of the LDPR via a different triple.

Reply:
It is explicit in the spec that the LDPRv cannot `ldp:contains` the LDPCv. We are making them children in the JCR context, but avoiding creating the `ldp:contains` connection for the LDPCv.

Reply:
Ah, okay - I don't know how the internals work very well.   I wonder why the spec folks didn't want the LDPCv to be a child of the LDPR. If it was connected to the life cycle, then this won't work either.   Did we decide that if a LDPR is deleted so is all it's version history?  I know we talked about it but can't recall what was decided.

Reply:
That is the current plan, everything is nested.
`LDPRv -> LDPCv -> LDPRm(s)`

But @lsitu noted that currently we turn off versioning by deleting the `LDPCv` which means you can't toggle versioning on a resource without destroying all previously made versions. So we may need to review that later.